requirements-docs.txt

```diff
@@ -486,4 +486,4 @@ ws4py==0.5.1
 
 # The following packages are considered to be unsafe in a requirements file:
 # setuptools
-python-apt @ https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/python-apt/2.4.0ubuntu1/python-apt_2.4.0ubuntu1.tar.xz ; sys.platform == "linux"
+python-apt @ https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/python-apt/2.4.0ubuntu1/python-apt_2.4.0ubuntu1.tar.xz ; sys.platform == "linux"
```

Comment:
Odd, I don't see what changed here

Reply:
Probably.  Not worth worrying about, just curious

Reply:
whitespace at the end of the line on the original